LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2020 LightStep
```

Comment:
ok I know I told you to put 2020 but did a bit more googling and it sounds like the most correct thing is for this to be the date when it was originally published, which it looks like was 2016: https://github.com/lightstep/lightstep-tracer-android/commits/master?after=683c98dc06b7720175e4c03913d06feb8ec963ca+454&branch=master